forward_velocity: 0.2
back_velocity: -0.2
back_time: 1
forward_angle: 0.35
rotation_velocity: 0.5
emergency_threshold: 0.1
road_center_threshold: 5
road_threshold: 1.5
curve_gain: 2
avoidance_gain: 0.4
road_center_gain: 0.8
wall_forward_angle: 0.35
wall_rate_threshold: 0.8
wall_distance_upper_threshold: 5
wall_distance_lower_threshold: 0.5
emergency_diff_threshold: 0.1
angle_bias: 0
path_back_interval: 5
goal_reset_rate: 1
costmap_margin: 0.4
esc_map_div_x: 3
esc_map_div_y: 3
esc_map_width: 0.9
esc_map_height: 0.9
safty_range_threshold: 1.5
safty_rate_threshold: 0.1
